tuxemon/menu/menu.py

```diff
@@ -494,7 +494,7 @@ def process_event(self, event: PlayerInput) -> Optional[PlayerInput]:
             handled_event = True
             if event.pressed and self.escape_key_exits:
                 self.close()
-
+    
```

Comment:
formatting :D

Reply:
Hopefully this lastest commit fixes the issue, lol.